lib/turn.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/turn'
+require './lib/card'
+require './lib/deck'
+require './lib/round'
+
+class Turn
+    attr_reader :cards, :turns
+
+  def initialize(cards, turns)
+    @cards = card
+    @turns = turn
+  end
```

Comment:
Go ahead back to the interaction pattern and take a look at the Turn Class interaction pattern. It shows that when a Turn object gets initialized, it takes in two arguments (which you have), but, it should be taking in a guess and the card that's being guessed on.

Reply:
You can initialize this class with the card and the guess, and then create attr_readers to be able to read the card and guess that get passed in.